wasp/boards/pinetime/watch.py.in

```diff
@@ -66,6 +66,7 @@ battery = Battery(
         Signal(Pin('USB_PWR', Pin.IN), invert=True))
 button = Pin('BUTTON', Pin.IN)
 i2c = I2C(1, scl='I2C_SCL', sda='I2C_SDA')
+accel = BMA421(i2c)
```

Comment:
Thanks for doing the merge (and I hope you like having step counting ;-) I'm super pleased to finally have that running). However... this looks like a by-hand merge which is pretty painful for me to work with and probably isn't simplest way for you either.

I don't to give an (unsolicited) git lecture so, keeping it short, for an unmerged patch I'd typically use git something like this:
~~~
git fetch origin
git branch backup-20200610
git rebase origin/master
# Fixup merge issues and git add the files that have been fixed
git rebase --continue
# Test ;-)
git push --force myrepo HEAD
~~~